lesson-17/script.js

```diff
@@ -0,0 +1,256 @@
+    dayPart.classList.add('daily_forecast_day');
+    dayForecast.append(dayPart);
+    
+    const maxTemperature = document.createElement('p');
+    maxTemperature.classList.add('daily_forecast_temp-max');
+    maxTemperature.innerText = `MAX temperature: ${getTempInCelsium(maxTemp)}°C`;
+    dayPart.append(maxTemperature);
+    
+    const dayPartClouds = document.createElement('p');
+    dayPartClouds.classList.add('daily_forecast_clouds-day');
+    dayPartClouds.innerText = `${dayClouds} is expected.`;
+    dayPart.append(dayPartClouds);
+
+    const dayPartIcon = document.createElement('img');
+    dayPartIcon.setAttribute('class', 'daily_forecast_day_icon');
+    dayPartIcon.setAttribute('src', weatherIcons.dayIcons[dayClouds]);
+    dayPartIcon.setAttribute('alt', dayClouds);
+    dayPartIcon.innerText = `${dayClouds} is expected.`;
+    dayPart.append(dayPartIcon);
+
+    const nightPart = document.createElement('div');
+    nightPart.classList.add('daily_forecast_night');
+    dayForecast.append(nightPart);
+    
+    const minTemperature = document.createElement('p');
+    minTemperature.classList.add('daily_forecast_temp-min');
+    minTemperature.innerText = `MIN temperature: ${getTempInCelsium(minTemp)}°C`;
+    nightPart.append(minTemperature);
+    
+    const nightPartClouds = document.createElement('p');
+    nightPartClouds.classList.add('daily_forecast_clouds-night');
+    nightPartClouds.innerText = `${nightClouds} is expected.`;
+    nightPart.append(nightPartClouds);
+
+    const nightPartIcon = document.createElement('img');
+    nightPartIcon.setAttribute('class', 'daily_forecast_night_icon');
+    nightPartIcon.setAttribute('src', weatherIcons.nightIcons[nightClouds]);
+    nightPartIcon.setAttribute('alt', nightClouds);
+    nightPartIcon.innerText = `${nightClouds} is expected.`;
+    nightPart.append(nightPartIcon);
+};
+
+function displayNeighborsList () {
+    const neighborsList = document.createElement('div');
+    neighborsList.classList.add('neighbors-container');
+    document.querySelector('.main').append(neighborsList);
+};
+
+function displayNeighborCity (cityName, locationKEY) {
+    const neighborCity = document.createElement('button');
+    neighborCity.innerText = cityName;
+    neighborCity.classList.add('neighbors-container_neighbor-city');
+    neighborCity.setAttribute('id', locationKEY);
+
+    document.querySelector('.neighbors-container').append(neighborCity);
+};
+
+document.querySelector('.header_city-select_btn').addEventListener('click', () => {
+    const selectedCity = document.querySelector('.header_city-selector');
+    const choosedCity = document.querySelector('.header_city');
+    choosedCity.innerText = selectedCity.value;
+
+    clearWeather();
+    searchCity(selectedCity.value);
+    clearSearch();
+});
+
+document.addEventListener('click', (event) => {
+    const element = event.target;
+    const choosedCity = document.querySelector('.header_city');
```

Comment:
Aslo uses twice, so can be moved to top level
```suggestion
    const chosenCity = document.querySelector('.header_city');
```